topics-and-advance-readings/identity-as-linked-data-on-immutable-ledgers.md

```diff
@@ -8,39 +8,36 @@ By
 
 
 ## The COALA IP Working Group
 
-To address these problems a group called COALA IP (Coalition Of Automated Legal Applications, Intellectual Property)
-formed to design and implement a free and open specification for handling digital licensing of intellectual property. It's
-goals are to establish open, free and easy ways to claim attribution, add meta data, license works, mediate IP disputes
-and authenticate claims of others, all in a way where there is a global agreement at the data level, without centralized
+To address these problems, COALA IP (Coalition Of Automated Legal Applications, Intellectual Property) was
+formed to design and implement a free and open specification for handling digital licensing of intellectual property. Its
+goals are to establish open, free, and easy ways to claim attribution, add metadata, license works, mediate IP disputes,
+and authenticate claims of others. Furthermore, there should be global agreement at the data level without the need for centralized
 control.
 
-A recent, more concrete, endeavor of the group has been to write a specification for a protocol handling digital licensing of
+A recent, more concrete, endeavor of the group has been to write a specification for handling digital licensing of
 intellectual property on immutable ledgers. It's an effort to transform the implementation-agnostic [Rights Reference Model](http://doi.org/10.1000/284)
-of the [Linked Content Coalition](http://www.linkedcontentcoalition.org/index.php) into a free and open guideline,
-outlining technologies that could be leveraged for implementation as well as structuration on how canonicalize an easily
-approachable specification for all involved parties (creators, rights holders, consumers, developers, ...). The protocol is
-ought to be technology-opinionated, while being ledger-agnostic.
+of the [Linked Content Coalition](http://www.linkedcontentcoalition.org/index.php) into a free and open guideline. It
+outlines technologies that could be leveraged for implementation and structure of a specification for all involved parties: creators, rights holders, consumers, developers, etc. The protocol is to be technology-opinionated, but ledger-agnostic.
 
 
 ## The COALA IP Protocol
 
-The COALA IP protocol is essentially two concurrent technical efforts:
+The COALA IP protocol is essentially two parallel technical efforts:
 
-1. It's a community-driven effort to find and define a minimal-viable set of data for licensing intellectual property
+1. It's a community-driven effort to find and define a minimally-viable set of data for licensing intellectual property
    (using RDF schema definitions)
 2. It's a free and open messaging/communication protocol for license-transactions
 
 
-At its core, the RDF schema defines six main entities that form an ontology by connecting to other respective entities. The
+At its core, the RDF schema defines ontology over six main entities that form and their interconnections. The
```

Comment:
Nope! I meant "... defines ontology over six main entities and their
interconnections"

On Tue, Jul 12, 2016 at 5:57 PM, Tim Daubenschütz <notifications@github.com>
wrote:

> In
> topics-and-advance-readings/identity-as-linked-data-on-immutable-ledgers.md
> <https://github.com/TimDaub/rebooting-the-web-of-trust/pull/1#discussion_r70468164>
> :
>
> >     (using RDF schema definitions)
> >  2. It's a free and open messaging/communication protocol for license-transactions
> >
> >
> > -At its core, the RDF schema defines six main entities that form an ontology by connecting to other respective entities. The
> > +At its core, the RDF schema defines ontology over six main entities that form and their interconnections. The
>
> Did you intend to fix the sentence in this way? Sounds kinda weird :D
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TimDaub/rebooting-the-web-of-trust/pull/1/files/7491408ab81e610a29020e4357f733fde3e2e276#r70468164>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGNzxB5P4knrAIIqGVa65ERocQecLU0Zks5qU7mCgaJpZM4JKdI2>
> .
>